README.md

```diff
@@ -44,6 +48,31 @@ npm i -D eslint-plugin-vue
 Example .eslintrc.js Vue setup:
 ```js
 module.exports = {
-  extends: ['eslint-config-digitalbazaar/vue']
+  extends: ['digitalbazaar/vue'] // only the vue rules and any rules in parent dirs
+}
+```
+
+To use the jsdoc rules you will need to install the eslint-plugin-jsdoc
```

Comment:
jsdoc is being referred to variously as `jsdoc` and `JSDoc`, make consistent throughout.

Reply:
Official name is JSDoc so using that.

Reply:
jsdoc is now JSDoc and vue is now Vue.